pkg/rtc/mediatrackreceiver.go

```diff
@@ -691,6 +689,28 @@ func (t *MediaTrackReceiver) GetAudioLevel() (float64, bool) {
 	return receiver.GetAudioLevel()
 }
 
+func (t *MediaTrackReceiver) RecordPublishRTPStats() {
+	t.lock.RLock()
+	defer t.lock.RUnlock()
+	for _, receiver := range t.receiversShadow {
+		if wr, ok := receiver.TrackReceiver.(*sfu.WebRTCReceiver); ok {
+			for layer := range receiver.layerSSRCs {
```

Comment:
if the track is not a simulcast encoding, the layerSSRC will be empty, can use layer 0 for that.